README.md

```diff
@@ -31,15 +42,14 @@ Except where noted, all samples build from the command line on
 any platform supported by .NET Core. There are a few samples that are
 specific to Visual Studio and require Visual Studio 2017 or later. In 
 addition, some samples show platform specific features and will require 
-a specific platform.
+a specific platform. Other samples and snippets require the .NET Framework
+and will run on Windows platforms, and will need Visual Studio to build.
```

Comment:
VS isn't required; the Developer Pack for a particular .NET Framework version is. (You can then compile with the command line.)